bigchaindb/commands/bigchain.py

```diff
@@ -157,8 +157,20 @@ def run_drop(args):
 def run_start(args):
     """Start the processes to run the node"""
     logger.info('BigchainDB Version {}'.format(bigchaindb.__version__))
+
     bigchaindb.config_utils.autoconfigure(filename=args.config, force=True)
 
+    if args.allow_temp_keypair:
+        if not (bigchaindb.config['keypair']['private'] or
+                                  bigchaindb.config['keypair']['public']):
+
+            private_key, public_key = crypto.generate_key_pair()
+            bigchaindb.config['keypair']['private'] = private_key
+            bigchaindb.config['keypair']['public'] = public_key
+        else:
+            logger.warning('Keypair found, no need to create one on the fly.')
```

Comment:
The logs should give to the user enough info about that ("→" mine):

```
INFO:bigchaindb.commands.bigchain:BigchainDB Version 0.6.0
→ INFO:bigchaindb.config_utils:Configuration loaded from `/home/vrde/.bigchaindb`
→ WARNING:bigchaindb.commands.bigchain:Keypair found, no need to create one on the fly.
INFO:bigchaindb.commands.bigchain:RethinkDB started with PID 2191
→ INFO:bigchaindb.commands.bigchain:Starting BigchainDB main process with public key Ajh1ehp4hYQGqgq5w8QhfNxn2pCHpGuNNbhm5p7gnjDn
```

is it enough in your opinion?